client/src/components/TodoForm.js

```diff
@@ -1,9 +1,9 @@
-import { useState } from 'react';
-import classes from './TodoForm.module.css';
+import { useState } from "react";
+import classes from "./TodoForm.module.css";
 
 const TodoForm = () => {
-  const [itemText, setItemText] = useState('');
-  const [error, setError] = useState('');
+  const [itemText, setItemText] = useState("");
+  const [error, setError] = useState("");
```

Comment:
Discuss coding styles with your team to prevent issues like this. single-quotes/double-quotes, semi colon etc